frappe/public/js/frappe/form/sidebar/assign_to.js

```diff
@@ -139,6 +139,26 @@ frappe.ui.form.AssignToDialog = class AssignToDialog {
 
 		me.dialog.set_value("assign_to", assign_to);
 	}
+	user_group_list() {
+		let me = this;
+		let user_group_members = [];
+
+		let user_group = me.dialog.get_value("assign_to_user_group");
+		me.dialog.set_value("assign_to_me", 0);
+
+		if (user_group) {
+			frappe.db.get_list("User Group Member", {
+				parent_doctype:"User Group",
+				filters:{"parent":user_group}, 
```

Comment:
Format the code. https://github.com/frappe/erpnext/wiki/Pull-Request-Checklist#linting